gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -562,6 +562,7 @@ public class PegasusPlugin implements Plugin<Project>
   private static final Pattern TEST_DIR_REGEX = Pattern.compile("^(integ)?[Tt]est");
   private static final String SNAPSHOT_NO_PUBLISH = "rest.model.noPublish";
   private static final String SNAPSHOT_FORCE_PUBLISH = "rest.model.forcePublish";
+  private static final String IDL_AUTO_GENERATION_ENABLED = "rest.idl.autoGeneration.enabled";
```

Comment:
just one minor comment otherwise looks good to me.

Why this word "autoGeneration" ? I cannot understand only by the term itself. I suggest either change to something more intuitive, or add a comment here.

Reply:
updated the config to processEmptyIdlDir